script/core/ActivateMarket.s.sol

```diff
@@ -39,29 +39,29 @@ contract ActivateMarket is Deployer {
     tkn1: first tokens
     tkn2: second tokens,
     tickSpacing: tick spacing,
-    tkn1_in_gwei: price of one tkn1 (display units) in gwei
-    tkn2_in_gwei: price of one tkn2 (display units) in gwei
+    tkn1_in_mwei: price of one tkn1 (display units) in mwei (1mwei = 1e-12 eth = 1e6 wei)
+    tkn2_in_mwei: price of one tkn2 (display units) in mwei 
     fee: fee in per 10_000
   */
 
   /* 
-    tknX_in_gwei should be obtained like this:
+    tknX_in_mwei should be obtained like this:
     1. Get the price of one tknX display unit in native token, in display units.
-       For instance, on ethereum, the price of 1 WETH is 1e9 gwei
-    2. Multiply by 1e9
+       For instance, on ethereum, the price of 1 WETH is 1e12 mwei
```

Comment:
This explanation doesn't make sense to me:
Line 49 says: Get the price __in display units__.
Line 50 says: The price of WETH is 1e12 mwei. - that's not *display units* 🤔 Shouldn't that be "1"?

Reply:
I would say "the price 1 WETH (in display units) (aka 1e18 WETH in wei) is 1e12 mwei", does that make sense?

Reply:
First, I would change the example to non-WETH since that makes the text somewhat ambiguous.

Second, I would only state the native token in display units in the example in Step 1. Step 2 deals with converting to mwei, so I'd continue the example there (instead of stuffing it all into Step 1).

Reply:
Tried to fix in 480c9e9bbe02e02ccf6ed5611170f5760db23de6